vtex-apps/organizations
# Adding an additional request to show business organization info.

#### What does this PR do? \*
Shows the business organization information that was missing.

#### How to test it? \*
Login with any user that has a Manager role, on the top menu select My Account->My Users and check if the name, email, telephone and address attributes are correctly showing on screen.

#### Describe alternatives you've considered, if any. \*



#### Related to / Depends on \*
